KISSmetricsAPI/KMAConnection.m

```diff
@@ -61,10 +61,39 @@ - (void)sendRecordWithURLString:(NSString *)urlString delegate:(id <KMAConnectio
     // Retain the urlString, we'll need to pass this back to the delegate
     _urlString = urlString;
     
-    // We use the default cachePolicy but set our own timeoutInterval
-    NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:urlString]
+    NSURL* url = [NSURL URLWithString:urlString];
+    
+    long query_length = (long) url.query.length;
+    
+    NSMutableURLRequest *request;
+    
+    if (query_length > 2000)
```

Comment:
While glancing at some other code, I noticed [this comment](https://github.com/kissmetrics/KISSmetrics-iOS-SDK/blob/master/KISSmetricsAPI/KISSmetricsAPI.h#L216-L222). Could we update/remove it, since it should no longer apply.